jboss-kie-kieserver/added/launch/kieserver-verify.sh

```diff
@@ -20,7 +20,7 @@ function verifyServerContainers() {
         done
         local containerVerifier="org.kie.server.services.impl.KieServerContainerVerifier"
         log_info "Attempting to verify kie server containers with 'java ${containerVerifier} ${releaseIds}'"
```

Comment:
Please add the JAVA_OPTS_APPEND in the log info

Reply:
why? I can't see a reason to do this.

Reply:
Could be useful to see in the place where is used instead to search in multi thousand line log the value, otherwise is more clear remove the part  'java' from the log if we omitting some args passed to java

Reply:
+1